third_party/move/move-compiler-v2/tests/bytecode-generator/v1-typing/extraneous_acquire.exp

```diff
@@ -0,0 +1,13 @@
+
+Diagnostics:
+error: unnecessary acquires annotation
```

Comment:
Why do we have `bytecode-gennerator\v1-typing` tests? Those tests don't seem to belong here.

Reply:
Moved to `acquires-checker/v1-tests`